# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hipblaslt/hotfix-libdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hipBLASLt-rocm-6.3.3/library/CMakeLists.txt.vanilla	2025-04-01 18:34:21.790419011 +0200
+++ hipBLASLt-rocm-6.3.3/library/CMakeLists.txt	2025-04-01 18:37:39.137551809 +0200
@@ -138,6 +138,8 @@
 # Target compile features
 target_compile_features(hipblaslt PRIVATE cxx_nullptr)
 
+set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -L/opt/rocm/lib64 -L/opt/rocm/lib")
+
 # Target include directories
 target_include_directories(hipblaslt
                            PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/include>
